addons_rksv/pos_rksv/models/res_company.py

```diff
@@ -10,6 +10,7 @@ class Company(models.Model):
     _name = 'res.company'
     _inherit = 'res.company'
 
+    rksv_at = fields.Boolean('RKSV AT', default=True)
```

Comment:
Right, this use case could be rational and there are other exceptions...so, I agree, forcing is not an option...the company running Odoo is responsible for that...maybe we should lock the setting in case an active PoS is registered for a company with BMF, so it can't happen it is turned off after it was setup as company for RKSV.